drivers/processor/processr/cpu.inf

```diff
@@ -121,7 +121,7 @@ LoadOrderGroup = Extended Base
 
 [Strings]
 
-ReactOS = "ReactOS Foundation"
+ReactOS = "ReactOS Project"
```

Comment:
In the INF files we're using "ReactOS Team" in all other places, so let's keep this in sync.
```suggestion
ReactOS = "ReactOS Team"
```
Example: https://github.com/reactos/reactos/blob/1225d4d3922343b6590e85f2c74d4edac8c61c6f/media/inf/usbport.inf#L119

Reply:
On the contrary, the idea of CORE-18191 is to have a single consistent naming everywhere to hopefully never deal with this negligible issue ever again.
I therefore like to see all occurrences of "ReactOS Team" being replaced with "ReactOS Project" as well.

Sorry for the confusion that happened here!

Reply:
We already have most translations for "ReactOS Team" in the INF files, this change would be breaking one. If we want to keep consistency so much, I think it would be better to do in a separate commit.

Reply:
I see. My point is: We currently put "ReactOS Team" or "ReactOS Foundation" where original Windows has "Microsoft Corporation".
That "Microsoft Corporation" string is never translated, and I want to establish "ReactOS Project" as the same consistent never-translated "company name". Sure, we have ReactOS Deutschland e.V., but even after 13 years, our association exists mostly next to the project whenever a legal entity is required (e.g. for the trademark), but it doesn't _own_ the ReactOS Project.

As such, it may have been a mistake in the first place to translate the ReactOS/ReactOS Team string.